ERCS/erc-201.md

```diff
@@ -0,0 +1,147 @@
+---
+eip: 201
+title: Token Standard
```

Comment:
```suggestion
title: Fungible Token with gas rebates
```
The title should succinctly describe the ERC.
Standard is superfluous in an ERC title